### ISSUE TITLE
About Training Loss

### ISSUE THREAD

**Issue by zzgnb** [FireRedTeam/FireRedASR#86]
Hey Author,​​

​Thanks for sharing your work! I was reading your paper, and saw that you used Cross-Entropy Loss based on the transcript for training. Just wanna check, did you also use CTC Loss to handle the alignment?​​
